### PR TITLE
Refactor conversion of None to empty introduced in #49 and #40

### PR DESCRIPTION
**Pull request by jashgala**
Handled refactoring discussed in Issue #52.

Moved check for None to TextUtils and updated the github.py functions to use that instead of directly checking in the function itself.

### CODE REVIEW

**Comment by donnemartin** [comment]
Hi @jashgala, thanks for the PR!  I hope to review it this week.

File: gitsome/utils.py [hunk 1]
```diff
@@ -184,3 +184,14 @@ def _safe_split(self, text):
             return words
         except:
             return text
+
+    def sanitize_if_none(self, text):
+        """Sanitizes text to ensure it is not None
```
Comment (donnemartin) [inline gitsome/utils.py]: Could you [PEP-257-ify](https://www.python.org/dev/peps/pep-0257/) this docstring to the following?

Sanitize text to ensure it is not None.

File: gitsome/utils.py [hunk 2]
```diff
@@ -184,3 +184,14 @@ def _safe_split(self, text):
...
+
+        :type text:str
+        :param text: String to sanitize
```
Comment (donnemartin) [inline gitsome/utils.py]: Also, a period here.

String to sanitize.

**Comment by donnemartin** [comment]
@jashgala sorry for the late review, thanks again for the PR.  

I've requested some minor tweaks to the docstring.  

Also, could you resolve the conflicts?  

Thanks!

**Comment by jashgala** [comment]
Continued in #100 

(Changed target branch)

**Pull request closed without merging**